src/main/groovy/life/qbic/example/Activator.groovy

```diff
@@ -0,0 +1,20 @@
+package life.qbic.example
+
+import org.osgi.framework.BundleActivator
+import org.osgi.framework.BundleContext
+
+class Activator implements BundleActivator{
+
+    @Override
+    void start(BundleContext context ) throws Exception {
+
+        System.out.println( context.getBundle().getSymbolicName() +  ": Hello OSGi-World." )
```

Comment:
are these and the car example classes needed?

Reply:
They're introduced by the QUBE Template, I'm unsure how to proceed here since we would have to delete them every time we merge TEMPLATE back into development, (when changes are made to the scaffolding code)